Gemfile.lock

```diff
@@ -341,16 +370,16 @@ GEM
     rack (1.6.4)
     rack-test (0.6.3)
       rack (>= 1.0)
-    rails (4.2.5.2)
-      actionmailer (= 4.2.5.2)
-      actionpack (= 4.2.5.2)
-      actionview (= 4.2.5.2)
-      activejob (= 4.2.5.2)
-      activemodel (= 4.2.5.2)
-      activerecord (= 4.2.5.2)
-      activesupport (= 4.2.5.2)
+    rails (4.2.6)
```

Comment:
that was unintentional.  though i think it was the result of `bundle install` when i added eye.  didn't investigate further, but can play around with it a bit before submitting for final review.